ztp/gitops-subscriptions/argocd/resource-hook-example/siteconfig/kustomization.yaml

```diff
@@ -1,4 +1,6 @@
 resources:
 - post-sync.yaml
 - pre-sync.yaml
-- test-sno.yaml  
+- test-sno.yaml
+- test-3node.yaml
+- test-standard.yaml
```

Comment:
In the current hook methodology there's no need for the kustomization file. No harm in adding these here but for now its a noop.